capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -200,6 +200,7 @@ function c4m_restful_quick_post_preprocess_html(&$vars) {
   }
   $vars['attributes_array']['ng-app'][] = 'c4mApp';
   $vars['attributes_array']['ng-controller'][] = 'MainCtrl';
+  $vars['attributes_array']['ng-cloak'][] = 'true';
```

Comment:
It's not working in ``preprocess_page``, Should i add it to the tpl or keep it in ``preprocess_html`` ?